pandas/tests/computation/test_eval.py

```diff
@@ -1890,6 +1891,27 @@ def test_negate_lt_eq_le(engine, parser):
         tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    "column",
+    DEFAULT_GLOBALS,
```

Comment:
```suggestion
    DEFAULT_GLOBALS.keys(),
```

I think we just want the column names as strings?

Reply:
1) The result of DEFAULT_GLOBALS is dict
```
{'Timestamp': <class 'pandas._libs.tslibs.timestamps.Timestamp'>, 'datetime': <class 'datetime.datetime'>, 'True': True, 'False': False, 'list': <class 'list'>, 'tuple': <class 'tuple'>, 'inf': inf, 'Inf': inf}
```
2) The result of DEFAULT_GLOBALS.keys() is list of dict keys
```
dict_keys(['Timestamp', 'datetime', 'True', 'False', 'list', 'tuple', 'inf', 'Inf'])
```

When run pytest test_eval_no_support_column_name, column name obtained is same when use 1) DEFAULT_GLOBALS  or  2) DEFAULT_GLOBALS.keys()

For this, I update to `DEFAULT_GLOBALS.keys()`